test/Image2.rb

```diff
@@ -982,212 +982,212 @@ def levelize_channel
 #     end
 
     def test_magnify
-        assert_nothing_raised do
-            res = @img.magnify
-            assert_instance_of(Magick::Image, res)
-            assert_not_same(@img, res)
-        end
+      assert_nothing_raised do
+        res = @img.magnify
+        assert_instance_of(Magick::Image, res)
+        assert_not_same(@img, res)
+      end
 
-        res = @img.magnify!
-        assert_same(@img, res)
+      res = @img.magnify!
+      assert_same(@img, res)
     end
 
     def test_map
-        map = Magick::Image.read('netscape:').first
-        assert_nothing_raised do
-            res = @img.map(map)
-            assert_instance_of(Magick::Image, res)
-            assert_not_same(@img, res)
-        end
-        assert_nothing_raised { @img.map(map, true) }
-        assert_raise(NoMethodError) { @img.map(2) }
-        assert_raise(ArgumentError) { @img.map(map, true, 2) }
-        assert_raise(ArgumentError) { @img.map }
-        map.destroy!
-        assert_raise(Magick::DestroyedImageError) { @img.map(map, true) }
+      map = Magick::Image.read('netscape:').first
+      assert_nothing_raised do
+        res = @img.map(map)
+        assert_instance_of(Magick::Image, res)
+        assert_not_same(@img, res)
+      end
+      assert_nothing_raised { @img.map(map, true) }
+      assert_raise(NoMethodError) { @img.map(2) }
+      assert_raise(ArgumentError) { @img.map(map, true, 2) }
+      assert_raise(ArgumentError) { @img.map }
+      map.destroy!
+      assert_raise(Magick::DestroyedImageError) { @img.map(map, true) }
     end
 
     def test_marshal
-        img =  Magick::Image.read(IMAGES_DIR+'/Button_0.gif').first
-        d = nil
-        img2 = nil
-        assert_nothing_raised { d = Marshal.dump(img) }
-        assert_nothing_raised { img2 = Marshal.load(d) }
-        assert_equal(img, img2)
+      img =  Magick::Image.read(IMAGES_DIR+'/Button_0.gif').first
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.